ONSdigital/design-system
# Add yarn command to build script

### What is the context of this PR?
Due to new packages being added since I last ran the build script there were errors for stylelint and prismjs not being found. Installing them using the yarn command fixed this

### How to review 
- Run `yarn build` locally on master and check for errors
- Run on this branch and see that there are no errors
